notifications/templates/email/local_units/local_unit.html

```diff
@@ -0,0 +1,43 @@
+<p>Dear {{full_name}} {% include "includes/logo.html" %}</p>
+
+{% if new_local_unit %}
+<p>A new local unit has been added and is awaiting your validation. Kindly review the information and validate or delete as necessary.</p>
+{% endif %}
+
+{% if update_local_unit %}
+<p>The local unit has been updated and is awaiting your validation. Kindly review the information and validate or delete as necessary.</p>
+{% endif %}
+
+{% if validate_success %}
+<p>
+    Your request for the local unit has been successfully validated. The local unit is now visible and marked as validated.
+</p>
+{% endif %}
+
+{% if revert_reason  %}
+<p>
+    Your local unit has been reverted.{{revert_reason}} Please review and resubmit if applicable.
+</p>
+{% endif %}
+
+{% if deprecate_local_unit %}
+<p>
+    Your local unit has been deprecated. {{deprecate_reason}} Please contact the administrator for more information.
+</p>
+{% endif %}
+
+{% if regional_admin %}
+<p>
+    The new local unit has been pending validation for over 14 days. Kindly review and validate or delete as necessary.
+</p>
+
+{% endif %}
+
+{% if global_admin %}
```

Comment:
Not sure if the variable name is correct here.

Reply:
They are passed through context variable from this file local_units/management/commands/notify_validators.py